packages/fx-core/src/plugins/solution/fx-solution/solution.ts

```diff
@@ -583,22 +583,6 @@ export class TeamsAppSolution implements Solution {
       return canProvision;
     }
 
-    const provisioned = this.checkWetherProvisionSucceeded(ctx.config);
```

Comment:
Simply removing state checking here doesn't seem to deliver "click provision again, will check resources and show success message"

Reply:
we will have a provision success message instead.